autorel/builders/debian_source_builder.py

```diff
@@ -0,0 +1,80 @@
+"""
+    module : debian_source_builder
+    This module is responsible for building debian source package
+    provided the distribution tarball directory
+"""
+import os
+import glob
+
+
+class DebianSourcePackage(object):
+    """
+        Result of running DebianSourceBuilder,
+        represents a Debian source package
+    """
+    def __init__(self, distribution_tarball, patch_file, source_control_file):
+        self._distribution_tarball = distribution_tarball
+        self._patch_file = patch_file
+        self._source_control_file = source_control_file
+
+    @property
+    def distribution_tarball_path(self):
+        return self._distribution_tarball
+
+    @property
+    def patch_file_path(self):
+        return self._patch_file
+
+    @property
+    def source_control_file_path(self):
+        return self._source_control_file
+
+
+
+class DebianSourceBuilder(object):
+    """
+        Responsible for creating debian source package from
+        the distribution tarball.
+    """
+    def __init__(self, input_directory, image_name):
+        self._input_directory = input_directory
+        self._image_name = image_name
```

Comment:
Yeah, but the `DebianSourceBuilder` has nothing to do with a Docker image... this is an implicit dependency.
What if I want to run it without Docker? It will work, but I have to pass an `image` to the constructor... (and the solution is not making the image optional :-) )